src/operator/subgraph/dnnl/dnnl_fc.cc

```diff
@@ -654,14 +651,14 @@ static bool SgMKLDNNAvoidFCQuantizeInput(const NodeAttrs& attrs,
   return avoid_indexes.count(index_to_check);
 }
 
-NNVM_REGISTER_OP(_sg_mkldnn_fully_connected)
-    .describe(R"code(_sg_mkldnn_fully_connected)code" ADD_FILELINE)
+NNVM_REGISTER_OP(_sg_dnnl_fully_connected)
```

Comment:
please add alias with the old name, it could be done in separate PR

Reply:
`_sg_dnnl_fully_connected` -> `_sg_onednn_fully_connected`